aloha/real_env.py

```diff
@@ -1,122 +1,113 @@
-            robot_name='follower_left',
-            node=node,
-            iterative_update_fk=False,
-        )
-        self.follower_bot_right = InterbotixManipulatorXS(
-            robot_model='vx300s',
-            group_name='arm',
-            gripper_name='gripper',
-            robot_name='follower_right',
-            node=node,
-            iterative_update_fk=False,
-        )
+        Initialize the Real Robot Environment.
+
+        :param node: The InterbotixRobotNode instance used for configuring and controlling the robot.
+        :param setup_robots: If True, sets up the robot arms during initialization, defaults to True.
+        :param setup_base: If True, configures the robot's base, defaults to False.
+        :param torque_base: If True, enables torque on the robot's base after setup. Only relevant if
+                            the base is mobile, defaults to False.
+        :param config: Dictionary of configuration parameters, including base type and robot settings.
 
-        self.recorder_left = Recorder('left', node=node)
-        self.recorder_right = Recorder('right', node=node)
-        self.image_recorder = ImageRecorder(node=node, is_mobile=IS_MOBILE)
+        :raises ValueError: Raised if setup_base is set to False but the robot base is expected to be mobile.
+        """
+        self.is_mobile = config.get('base', False)
+
+        self.DT = 1 / config.get('fps', 30)
+
+        # Dynamically import module based on config value
+        if self.is_mobile:
+            xs_robot_module = importlib.import_module(
+                'interbotix_xs_modules.xs_robot.slate')
+            self.InterbotixSlate = getattr(xs_robot_module, 'InterbotixSlate')
+
+        # Dictionary to store the robot instances
+        self.robots = {}
+
+        # Iterate through leader arms from the YAML
+        for leader in config.get('leader_arms', []):
+            self.robots[leader['name']] = InterbotixManipulatorXS(
+                robot_model=leader['model'],
+                robot_name=leader['name'],
+                node=node,
+                iterative_update_fk=False,
+            )
+
+        # Iterate through follower arms from the YAML
+        for follower in config.get('follower_arms', []):
+            self.robots[follower['name']] = InterbotixManipulatorXS(
+                robot_model=follower['model'],
+                group_name='arm',   # Assuming this remains the same for all followers
+                gripper_name='gripper',   # Assuming this remains the same for all followers
+                robot_name=follower['name'],
+                node=node,
+                iterative_update_fk=False,
+            )
+
+        self.follower_bots = [robot for name,
+                              robot in self.robots.items() if 'follower' in name]
+
+        self.follower_robots = {name: robot for name,
+                                robot in self.robots.items() if 'follower' in name}
```

Comment:
dict and list?

Reply:
I am trying to figure out a better solution for this.

Reply:
I have shifed it to the method where it is required. But I guess there is a more elegant way to handle it.